tempest/frm_userdata.sh

```diff
@@ -30,7 +30,15 @@ else
     apt-get update
     apt-get install qemu-guest-agent
     systemctl enable qemu-guest-agent
-    sed -i '/DAEMON_ARGS=/c DAEMON_ARGS="-F/etc/qemu/fsfreeze-hook"' /etc/init.d/qemu-guest-agent
+    path=$(systemctl status qemu-guest-agent | grep Loaded)
```

Comment:
Instead of service path, better to check on ubuntu version and add the steps according to the Ubuntu version.

Reply:
changed